<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QDialog" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Composition Helper</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="maximumSize">
         <size>
          <width>425</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-left: 30px;</string>
           </property>
           <property name="text">
            <string>Style</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="4">
          <widget class="QLabel" name="lblOptions">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Options</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="4">
          <widget class="QCheckBox" name="cbForceGR">
           <property name="toolTip">
            <string>When checked, constrain helper to respect Golden number ratio</string>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-left: 30px;</string>
           </property>
           <property name="text">
            <string>Force golden ratio</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="4">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Line properties</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-left: 30px;</string>
           </property>
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="QSlider" name="hsLineWidth">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Line width applied to helper</string>
           </property>
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>2500</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="pageStep">
            <number>500</number>
           </property>
           <property name="value">
            <number>200</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="14" column="0" colspan="4">
          <widget class="QPushButton" name="pbAdd">
           <property name="toolTip">
            <string>Add current helper configuration to document</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QDoubleSpinBox" name="dsbLineWidth">
           <property name="toolTip">
            <string>Line width applied to helper</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="suffix">
            <string>px</string>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>25.000000000000000</double>
           </property>
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="3">
          <widget class="CHWColorButton" name="pbLineColor">
           <property name="toolTip">
            <string>Color applied to helper</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="4">
          <widget class="QCheckBox" name="cbFlipH">
           <property name="toolTip">
            <string>Flip helper horizontally (vertical axis)</string>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-left: 30px;</string>
           </property>
           <property name="text">
            <string>Flip horizontally</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="10" column="0" colspan="4">
          <widget class="QCheckBox" name="cbFlipV">
           <property name="toolTip">
            <string>Flip helper vertically (horizontal axis)</string>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-left: 30px;</string>
           </property>
           <property name="text">
            <string>Flip vertically</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QComboBox" name="cbxHelpers">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>104</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Model of helper to use&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="3">
          <widget class="QComboBox" name="cbxLineStyle">
           <property name="toolTip">
            <string>Line style applied to helper</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Helper</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-left: 30px;</string>
           </property>
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0" colspan="4">
          <widget class="QCheckBox" name="cbUseSelection">
           <property name="toolTip">
            <string>When checked, if a selection exist, use selection bounds to render helper</string>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-left: 30px;</string>
           </property>
           <property name="text">
            <string>Use current selection</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblPreview">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>(preview)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbAbout">
        <property name="text">
         <string>About</string>
        </property>
        <property name="icon">
         <iconset theme="help-about"/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="toolTip">
         <string>Close &lt;i&gt;Composition Helper&lt;/i&gt; window</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset theme="dialog-close">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CHWColorButton</class>
   <extends>QPushButton</extends>
   <header>compositionhelper.ch.chwcolorbutton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
